docs/source/ubuntu-opens-install.rst

```diff
@@ -292,14 +328,10 @@ And/or for IPv6 connectivity between compute hosts:
    configuration appropriately. You should consult the relevant
    documentation for your chosen BGP stack.
 
-9.  Create the ``/etc/calico/felix.cfg`` file by taking a copy of the
-    supplied sample config at ``/etc/calico/felix.cfg.example``. Then,
-    in ``/etc/calico/felix.cfg``:
-
-    -  Change both the ``PluginAddress`` and ``ACLAddress`` settings to
-       the host name or IP address of the controller node.
+14.  Create the ``/etc/calico/felix.cfg`` file by taking a copy of the
```

Comment:
I think it's worth mentioning that you can specify the FelixHostname used, which must match nova's hostname for that box.